browser/brave_news/brave_news_controller_factory.cc

```diff
@@ -60,7 +60,7 @@ BraveNewsControllerFactory::~BraveNewsControllerFactory() = default;
 
 KeyedService* BraveNewsControllerFactory::BuildServiceInstanceFor(
     content::BrowserContext* context) const {
-  if (!brave::IsRegularProfile(context)) {
+  if (!brave::IsRegularProfile(context) || context->IsOffTheRecord()) {
```

Comment:
you shouldn't need this check as `GetBrowserContextToUse` default version is to NOT create a service in Incognito.